<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Index</class>
 <widget class="QMainWindow" name="Index">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="smallButon">
        <property name="styleSheet">
         <string notr="true">QToolButton{
	background-color: rgba(0, 0, 0,0);
    border:none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/imgs/suoxiao.svg</normaloff>:/imgs/suoxiao.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="closeButton">
        <property name="styleSheet">
         <string notr="true">QToolButton{
	background-color: rgba(0, 0, 0,0);
    border:none;
}
QToolButton#closeButton:hover{
	background-color: rgb(255, 68, 6);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/imgs/icons8-close.svg</normaloff>:/imgs/icons8-close.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="minimumSize">
         <size>
          <width>374</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="addToolButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/imgs/+.svg</normaloff>:/imgs/+.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="messageButton">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
image: url(:/imgs/xiaoxi.svg);
background-color: rgba(0, 0, 0,0);
border:none
}

QPushButton:hover{
	background-color: rgb(135, 130, 133);
    border:none
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="friendButton">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	image: url(:/imgs/lianxiren.svg);
    background-color: rgba(0, 0, 0,0);
    border:none;
}
QPushButton:hover{
	background-color: rgb(135, 130, 133);
    border:none
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="setButton">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton{
image: url(:/imgs/shezhi.svg);
background-color: rgba(0, 0, 0,0);
border:none
}
QPushButton:hover{
	background-color: rgb(135, 130, 133);
    border:none
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListView" name="listView">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>600</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>450</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{
	background-color: rgba(0, 0, 0,0);
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
